lib/project-data.ts

```diff
@@ -87,6 +97,25 @@ export class ProjectData implements IProjectData {
 		this.$errors.fail("No project found at or above '%s' and neither was a --path specified.", projectDir || this.$options.path || currentDir);
 	}
 
+	private getProjectIdentifiers(data :any): IProjectIdentifier {
+		let identifier: IProjectIdentifier;
+		data = data || "";
+
+		if (typeof data === "string") {
+			identifier = {
+				android: data,
+				ios: data
+			};
+		} else {
```

Comment:
What's the case when the android and ios package identifiers should be set to an empty string? Shouldn't we throw an error/warning here?